### PR TITLE
Basic request functionality

### PR DESCRIPTION
**Pull request by rhiannonruth**
connects #6

### CODE REVIEW

File: app/models/data_mapper_setup.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require_relative 'space'
+require_relative 'request'
```
Comment (hawksdoves) [inline app/models/data_mapper_setup.rb:2]: should require data_mapper and dm-postgres-adapter here and delete from models

File: app/models/data_mapper_setup.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 
 DataMapper.setup(:default,  ENV['DATABASE_URL'] || "postgres://localhost/makersbnb_#{ENV['RACK_ENV']}")
```
Comment (hawksdoves) [inline app/models/data_mapper_setup.rb:4]: Currently, we do not need ENV['DATABASE_URL'] ||

File: app/models/request.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'data_mapper'
```
Comment (hawksdoves) [inline app/models/request.rb:1]: see above- dry out (moving to data_mapper_setup)

File: spec/factories.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
```
Comment (hawksdoves) [inline spec/factories.rb:1]: Can you explain what factorygirl is and does. Why do we need to use it?

File: app/views/request/new.erb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<form action="/request/new" method="post">
```
Comment (hawksdoves) [inline app/views/request/new.erb:1]: This might not be needed now as we have the calendar functionality, so we do not need to manually input dates.

**Comment by rhiannonruth** [comment]
Covered in #5 - need to make sure we separate issues better in future.

**Pull request closed without merging**